### PR TITLE
Edit fix

### LINKED ISSUE #2: Feature request: Edit/delete reminder
As of the current version, I'm not able to edit, view, or easily delete a reminder. It would be a nice feature.

### PR DESCRIPTION
**Pull request by senpl**
This makes edit works, tested with limited cases(but with all types)
Code got many duplication with addTask, but I do not know flutter enough to make it more object oriented. Anyway I guess it is functional and could be improved over time. At this time this seems enough. Fix https://github.com/SamarthDesai01/RepeatMe/issues/2 in simple way. Also fix some details in addTasks(like inactive code). All sugestions or discussions are welcomed.

### CODE REVIEW

**Comment by senpl** [comment]
I suggest to change name from "Number" to "No of days" as more descriptive to what actually happens in app.

**Comment by senpl** [comment]
@SamarthDesai01 any progress with this merge ??? people want this feature.

**Comment by SamarthDesai01** [comment]
Hey sorry I didn't get back to you earlier! I've been swamped with work recently, but I'm hoping to take a proper look at it this weekend!

File: lib/views/add_task.dart [hunk 1]
```diff
@@ -427,7 +425,7 @@ class _AddTaskState extends State<AddTask> {
                   ),
                   new ChoiceChip(
                     //Number Chip
-                    label: Text('Number'),
+                    label: Text('No of Days'),
```
Comment (SamarthDesai01) [inline lib/views/add_task.dart]: Needs to be changed to "No. of Days" to be grammatically correct

**Review by SamarthDesai01 — CHANGES_REQUESTED** [review]
The label should read "No. of Days", everything else looks great though!

File: lib/views/add_task.dart (hunk 1 above)
Comment (senpl) [reply lib/views/add_task.dart]: I updated add_task and edit_task to "No. of Days". Commits below.

**Pull request merged**